### PR TITLE
Dovetail elixir

### PR DESCRIPTION
**Pull request by cavis**
Setup [dovetail-elixir](https://github.com/PRX/dovetail-elixir) app under the Dovetail ALB.

- [x] Add service/task-def/target-group for dovetail-elixir
- [x] Add listener rules to the dovetail ALB

Will need to merge PRX/dovetail-elixir#7 first.

### CODE REVIEW

File: stacks/dovetail.prx.org.yml [hunk 1]
```diff
@@ -124,6 +129,30 @@ Resources:
...
+  DovetailALBHTTPListenerRule:
+    Type: "AWS::ElasticLoadBalancingV2::ListenerRule"
+    Properties:
+      Actions:
+        - TargetGroupArn: !Ref DovetailApplicationALBTargetGroup
+          Type: forward
+      Conditions:
+        - Field: host-header
+          Values:
+            - dovetail.*
```
Comment (cavis) [inline stacks/dovetail.prx.org.yml]: IIRC, you get charged based on how many listener rules a request goes through.  So this sets up any `dovetail.prxu.org` requests as the first rule.  Then `dovetail3.*` requests.  Then the default (also the old Dovetail node app).

File: stacks/dovetail.prx.org.yml [hunk 2]
```diff
@@ -289,7 +354,7 @@ Resources:
       DeploymentConfiguration:
         MaximumPercent: 200
         MinimumHealthyPercent: 50
-      DesiredCount: !GetAtt DovetailServiceDesiredCountRolloverResource.DesiredCount
+      DesiredCount: 10
```
Comment (cavis) [inline stacks/dovetail.prx.org.yml]: Per the exchange-worker autoscaling PR, you no longer have to worry about this blowing away any autoscaling changes to the desired count.

File: stacks/dovetail.prx.org.yml [hunk 3]
```diff
@@ -350,72 +415,98 @@ Resources:
...
-      ServiceToken: !GetAtt DovetailServiceDesiredCountRolloverLambdaFunction.Arn
+      RetentionInDays: 14
+  DovetailElixirTaskDefinition:
+    Type: "AWS::ECS::TaskDefinition"
+    Properties:
+      ContainerDefinitions:
+        - Cpu: !Ref ContainerCpu
+          Environment:
+            - Name: APP_NAME
+              Value: "dovetail-elixir"
```
Comment (cavis) [inline stacks/dovetail.prx.org.yml]: I _think_ you can set the app-name as whatever you want in aws-secrets ... even including dashes.

File: stacks/dovetail.prx.org.yml (hunk 2 above)
Comment (farski) [reply stacks/dovetail.prx.org.yml]: Want to remove that rollover custom resource in this same PR?

Comment (farski) [reply stacks/dovetail.prx.org.yml]: JK you did

**Pull request merged**